<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemappedColourPaletteWidget</class>
 <widget class="QWidget" name="RemappedColourPaletteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="palette_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_28">
      <item>
       <widget class="QLabel" name="palette_label">
        <property name="text">
         <string>Palette:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="palette_filename_placeholder_widget" native="true"/>
      </item>
      <item>
       <widget class="QToolButton" name="select_palette_filename_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Choose CPT File</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="use_default_palette_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Use Default Palette</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../qt-resources/qt_widgets.qrc">
          <normaloff>:/tango_undo_16.png</normaloff>:/tango_undo_16.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="extra_placeholder_widget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="colour_scale_placeholder_widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="QGroupBox" name="range_mapping_group_box">
         <property name="title">
          <string>Palette Range</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_17">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="margin">
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="range_check_box">
            <property name="text">
             <string>Remap range</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="range_widget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>3</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="min_widget" native="true">
               <layout class="QHBoxLayout" name="scalar_range_min_layout_2">
                <property name="spacing">
                 <number>3</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="min_label">
                  <property name="text">
                   <string>Lower:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="min_line_edit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="max_widget" native="true">
               <layout class="QHBoxLayout" name="scalar_range_max_layout_2">
                <property name="spacing">
                 <number>3</number>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="max_label">
                  <property name="text">
                   <string>Upper:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="max_line_edit">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>80</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="range_restore_mean_deviation_group_box">
               <property name="title">
                <string>Data mean -/+ dev</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_18">
                <property name="spacing">
                 <number>3</number>
                </property>
                <property name="margin">
                 <number>3</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout2">
                  <property name="spacing">
                   <number>3</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="range_restore_mean_deviation_label">
                    <property name="text">
                     <string>Spread:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="range_restore_mean_deviation_spin_box">
                    <property name="keyboardTracking">
                     <bool>false</bool>
                    </property>
                    <property name="decimals">
                     <number>3</number>
                    </property>
                    <property name="minimum">
                     <double>0.010000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>1000.000000000000000</double>
                    </property>
                    <property name="singleStep">
                     <double>0.200000000000000</double>
                    </property>
                    <property name="value">
                     <double>2.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="range_restore_mean_deviation_button">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="toolTip">
                     <string>Use data mean +/- (spread * standard_deviation)</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="icon">
                     <iconset resource="../qt-resources/qt_widgets.qrc">
                      <normaloff>:/tango_undo_16.png</normaloff>:/tango_undo_16.png</iconset>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="range_restore_min_max_widget" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_29">
                <property name="spacing">
                 <number>3</number>
                </property>
                <property name="margin">
                 <number>3</number>
                </property>
                <item>
                 <widget class="QLabel" name="range_restore_min_max_label">
                  <property name="text">
                   <string>Data min-&gt;max:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="range_restore_min_max_button">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="toolTip">
                   <string>Use data min/max</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../qt-resources/qt_widgets.qrc">
                    <normaloff>:/tango_undo_16.png</normaloff>:/tango_undo_16.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qt-resources/qt_widgets.qrc"/>
 </resources>
 <connections/>
</ui>
